### ISSUE TITLE
On the fly category importer not useable for translation when language = store view

### ISSUE THREAD

**Issue by mblarsen** [dweeves/magmi-git#462]
In the situation where store views are used as a language category importer (as it is now) cannot be used to translate categories. 

@dweeves and others have written that you'll need to create a root and associate it to a store (not store view) through store management. But this is not possible with store views. They use whatever category root of what store they belong to.

```
store, sku, categories
world_english, NY99, [world_english]/men/shoes
world_danish, NY99, [world_danish]/mænd/sko
```
![store-view](https://cloud.githubusercontent.com/assets/247048/16804439/691d8098-493e-11e6-9306-277c4a7c8e75.png)

This gives the error

On the fly category creator/importer v0.2.5 - Cannot find site root with names:
world_danish

I kind of like how custom swatches work:

```
sku, store, color
NY99, world_danish, rød::[red]
```
The admin name is mentioned in `[]` at the end

Is something like this possible with categories as well since having multiple roots in Magento for store views is not possible?

E.g.

```
store, sku, categories
world_english, NY99, /men/shoes
world_danish, NY99, /mænd::1::1::0::[men]/sko::1::1::0::[shoes]
```

So basically it is the another options like active after the name of the category. It will be a translation rather than a creation of a new tree.
